File: regex/testers/tester.py
from sys import argv
from re import search

def file_to_str(filename):
    with open(filename) as file:
        lines = [line for line in file]
    return ''.join(lines)

target = file_to_str(argv[1])
regex = file_to_str(argv[2])

if search(regex, target) == None:
    print('False')
else:
    print('True')

File: regex/testers/tester.pl
use warnings;
use strict;

open my $targetfh, '<', $ARGV[0] or die "Can't open file $!";
open my $regexfh, '<', $ARGV[1] or die "Can't open file $!";

read $targetfh, my $target, -s $targetfh;
read $regexfh, my $regex, -s $regexfh;

if($target =~ /$regex/) {
	print("True\n");
}
else {
	print("False\n");
}
